configuration/curl/redirect-success-page.sh

```diff
@@ -7,6 +7,6 @@ curl -v --request POST \
            "amount":{"value":1000,"currencyCode":"GBP"},
            "reference":"Order-0001",
            "config": {
-               "redirectUrl": "http://site.com/checkout/success_pay"
+               "redirectUrl": "http://mymerchantsite.com/checkout/success_pay"
```

Comment:
To be consistent, we need to make the same changes for all code samples :)

Reply:
I don't think having the same example url is really important for consistency, however we should remove urls pointing to something real for sure.